README.md

```diff
@@ -1,4 +1,84 @@
-# tuned-lens
+# Tuned Lens 🔎
 Tools for understanding how transformer predictions are built layer-by-layer
 
 <img width="1028" alt="tuned lens" src="https://user-images.githubusercontent.com/39116809/206883419-4fb9083d-3fa0-48e9-ba97-b70cb21b08e9.png">
+
+This package provides a simple interface training and evaluating __tuned lenses__. A tuned lens allows us to peak at the iterative computations that a transformer is using the compute the next token.
+
+A lens into a transformer with n layers allows you to replace the last $m$ layers of the model with an [affine transformation](https://pytorch.org/docs/stable/generated/torch.nn.Linear.html) (we call these affine adapters).
```

Comment:
at the request of @StellaAthena we're now calling the adapters "translators" to prevent confusion with https://github.com/adapter-hub/adapter-transformers...